src/pudl/transform/eia.py

```diff
@@ -1083,6 +1083,73 @@ def _restrict_years(
     return df
 
 
+def map_balancing_authority_names_to_codes(df: pd.DataFrame) -> pd.DataFrame:
+    """Build a map of the BA names to their most frequent codes.
```

Comment:
"a mapping from" here don't make any sense to me/makes this more confusing to me.

Reply:
I went with `Build a map of the BA names to their most frequently associated BA codes.`

Reply:
Hmm. Is it because there's ambiguity between "building from" and "constructing out of?" I mean that the mapping you're creating maps **from** names **to** codes.